video/decode/vd_lavc.c

```diff
@@ -243,6 +243,8 @@ const struct autoprobe_info hwdec_autoprobe_info[] = {
     {"vdpau-copy",      HWDEC_FLAG_AUTO | HWDEC_FLAG_WHITELIST},
     {"mmal",            HWDEC_FLAG_AUTO},
     {"mmal-copy",       HWDEC_FLAG_AUTO | HWDEC_FLAG_WHITELIST},
+    {"mediacodec",      HWDEC_FLAG_AUTO},
+    {"mediacodec-copy", HWDEC_FLAG_AUTO | HWDEC_FLAG_WHITELIST},
```

Comment:
How did mediacodec work at all when not in this list? By always being explicitly chosen by mpv-android?

Reply:
Yes. With `hwdec=auto` it also just _happened_ to be at the top of the list.